libs/net/entities/ListOption.cs

```diff
@@ -0,0 +1,17 @@
+namespace TNO.Entities;
+
+/// <summary>
+/// Provides value types which control what users can enter for data.
+/// </summary>
+public enum ListOption
+{
+    /// <summary>
```

Comment:
check comment below about a `default` value

Reply:
Would nullable be a better option in this case?  Or should we stick to a default option which is essentially NA.

Reply:
The default behavior we have now is that every media type, list sources. So I wouldn't like to make it nullable, since when I will handle the code to load Source or Program (or another future option if that's the case) I won't have to deal with a null state. )That's why I would choose source as default just to keep the compatibility with the actual behavior after running a migration.

Reply:
If it works I'm in favour.